README.md

```diff
@@ -10,16 +14,64 @@
 [![python version](https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10%20%7C%203.11-blue)](https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10%20%7C%203.11-blue)
 [![scikit-learn supported versions](https://img.shields.io/badge/sklearn-1.0%20%7C%201.1%20%7C%201.2%20%7C%201.3-blue)](https://img.shields.io/badge/sklearn-01.0%20%7C%201.1%20%7C%201.2%20%7C%201.3-blue)
 
-With Intel(R) Extension for Scikit-learn you can accelerate your Scikit-learn applications and still have full conformance with all Scikit-Learn APIs and algorithms. This is a **free software AI accelerator** that brings over **10-100X** acceleration across a variety of applications. And you do not even need to change the existing code!
+---
+</h3>
+
+<div align="left">
+
+## Overview
+
+Intel(R) Extension for Scikit-learn is a **free software AI accelerator** designed to deliver up to **100X** faster performance for your existing scikit-learn code.
+The software acceleration is achieved with vector instructions, AI hardware-specific memory optimizations, threading, and optimizations for all upcoming Intel(R) platforms at launch time.
+
+
+With Intel(R) Extension for Scikit-learn, you can:
+
+* Speed up training and inference by up to 100x with the equivalent mathematical accuracy
+* Benefit from performance improvements across different Intel(R) hardware configurations
+* Integrate the extension into your existing Scikit-learn applications without code modifications
+* Continue to use the open-source scikit-learn API
+* Enable and disable the extension with a couple of lines of code or at the command line
+
+Intel(R) Extension for Scikit-learn is also a part of [Intel(R) AI Tools](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-analytics-toolkit.html). 
+
+## Installation
```

Comment:
Would go with API example first and then talk about installation/integration. 

The hypothesis is that we need to explain how to use this and what is benefit of using tool and then guide how to install. 

If the only user question is how to install this - then we already have buy-in and moving install instruction to later place will not impact conversion. 

p.s. - can we track any info from readme page? magic pixel and all that stuff?

Reply:
Good point! Changed the structure.